plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -104,6 +104,19 @@ public JdbcTableHandle(
         this.nextSyntheticColumnId = nextSyntheticColumnId;
     }
 
+    public JdbcTableHandle intersectedWithConstraint(TupleDomain<ColumnHandle> newConstraint)
```

Comment:
nit: separate commit

Reply:
It would be unused method in separate commit, so I'll prefer to keep it here.